services/ingestion/etl/fts/xls/test/stubs/record.json

```diff
@@ -0,0 +1,37 @@
+{
+  "Reference of the Legal Commitment (LC)": "SI2.462638",
+  "Commitment position key": "SI2.765808.1",
+  "Year": "2017",
+  "Name of beneficiary":
```

Comment:
This is one hell of a name 😆

Reply:
Not great indeed, the goal is to aggregate the information per-field and let the transform handle multi-value cases as in others.